scss/_footer.scss

```diff
@@ -1,167 +1,163 @@
-.footer .navbar-nav {
-  --#{$prefix}nav-link-font-weight: #{$font-weight-bold};
+.footer > * {
+  --#{$prefix}footer-part-title-font-size: #{$footer-font-size-md};
+  --#{$prefix}footer-part-title-font-weight: #{$footer-part-title-font-weight};
+  --#{$prefix}footer-part-title-line-height: #{$footer-line-height-md};
+  --#{$prefix}footer-part-title-letter-spacing: #{$footer-letter-spacing};
+  --#{$prefix}navbar-nav-font-size: #{$footer-font-size-sm};
```

Comment:
Why are they called `--#{prefix}navbar-nav` and not `footer-something`?

Reply:
Because it changes the navbar variable directly but it might be not clear, what would you preconize ?